packages-exp/messaging-exp/src/api.ts

```diff
@@ -49,13 +49,13 @@ export function getMessaging(app: FirebaseApp = getApp()): FirebaseMessaging {
 }
 
 /**
- * Subscribes the messaging instance to push notifications. Returns an FCM registration token
+ * Subscribes the Messaging instance to push notifications. Returns an FCM registration token
```

Comment:
I think `Messaging` is a literal here, correct?  That's the object name?  If so, we should backtick it everywhere we are using it as a literal, here and throughout.